plugins/ext-plugin/README.md

```diff
@@ -0,0 +1,15 @@
+# Ports Plug-in
```

Comment:
It would be good to add just a couple of sentences about what is this Plug-in for and an example how to use it.

Reply:
Sure, thanks for the report. I'll add how to use it in Vs code extensions

Reply:
I've updated README

Reply:
thx!